### PR TITLE
UI

### PR DESCRIPTION
**Pull request by UsirusenIkon**
In this PR, we did the following:

### Create a app.rb file that will serve as your console app entry-point. It should have methods that do the following:
- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person id.

### In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:
- Present the user with a list of options to perform.
- Lets users choose an option.
- If needed, ask for parameters for the option.
- Have a way to quit the app.

### CODE REVIEW

**Review by rustam-tolipov — COMMENTED** [review]
Hi @UsirusenIkon, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Congratulations on the effort you've put into this project! We appreciate it and know that great progress has been made, however we need to make a small adjustment before we can move forward with the review. Please remove or replace the Microverse logo from your readme file if you have a personal logo available for use. Once these steps are completed, we can move forward with the review process. Thank you for your understanding!




Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by UsirusenIkon** [comment]
please @Rustamxon7, you said, " Please remove or replace the Microverse logo from your readme file **if** you have a personal logo available for use". I do not have a personal logo for use.

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+  def create_book
+    puts 'Title:'
+    title = gets.chomp
+    puts 'Author:'
+    author = gets.chomp
+    book = Book.new(title, author)
+    @books << book
+    puts 'Book created'
+  end
```
Comment (FrederickMih) [inline app.rb]: - [ ] Let's kindly fix :wrench: the error on the screenshot below. Your app breaks when I tried to create a book.

Screenshot from 2023-01-06 22-02-28

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @UsirusenIkon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- PR is professional :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- All Linter checks passed :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rustam-tolipov** [comment]
> please @Rustamxon7, you said, " Please remove or replace the Microverse logo from your readme file **if** you have a personal logo available for use". I do not have a personal logo for use.

If then you are free to remove it!

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @UsirusenIkon👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**